packages/neos-ui/src/manifest.dataloaders.js

```diff
@@ -264,6 +264,12 @@ manifest('main.dataloaders', {}, globalRegistry => {
 
         _loadDataSourcesByOptions(options) {
             const cacheKey = makeCacheKey('', options);
+            const disableCaching =
+                options.dataSourceAdditionalData ?
+                    hasOwnProperty.call(options.dataSourceAdditionalData, 'disableCaching') ?
+                        options.dataSourceAdditionalData.disableCaching :
+                        false :
+                    false;
```

Comment:
Hey,

thanks for your PR :-)

Is there a reason you are putting this into "dataSourceAdditionalData" and not just into a top-level option? (Because according to the docs, dataSourceAdditionalData is sent to the server).

Furthermore, please add the option to the docs above.

Thanls :)

Reply:
I changed it to top-level option. This have more sense